assets/css/app.css

```
@@ -139,6 +139,12 @@
   }
 }
 
+@media (any-pointer: coarse) and (max-width: 1290px) {
```

Follow-up to my comment
```suggestion
@media (pointer: coarse) and (max-width: 1290px) {
```